hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -250,4 +259,37 @@ public static void checkRequiredProperties(TypedProperties props, List<String> c
     checkPropNames.forEach(prop ->
         Preconditions.checkState(!props.containsKey(prop), "Required property " + prop + " is missing"));
   }
+
+  /**
+   * Initialize the table if it does not exist.
+   *
+   * @param conf the configuration
+   * @throws IOException if errors happens when writing metadata
+   */
+  public static void initTable(Configuration conf) throws IOException {
+    final String basePath = conf.getString(FlinkOptions.PATH);
```

Comment:
IMO, `FlinkOptions.BASE_PATH` is more readable?

Reply:
I want to keep it sync with Spark data source, e.g. `org.apache.hudi.DefaultSource` line 62.